test/Integration/SqlOutputBindingIntegrationTests.cs

```diff
@@ -123,6 +123,7 @@ public void AddProductArrayTest(SupportedLanguages lang)
 
         [Theory]
         [SqlInlineData()]
+        [UnsupportedLanguages(SupportedLanguages.JavaScript)]
```

Comment:
For the Collectors test, since it's chsarp language specific, I can add that. Can you help me on what to add for the QueryTrigger and TimerTrigger tests, am not sure what to add except they aren't valid tests for JavaScript.

Reply:
Why aren't they valid? They both support JS as well (I'm assuming you meant QueueTrigger, not QueryTrigger) :

 https://docs.microsoft.com/en-us/azure/azure-functions/functions-bindings-timer?tabs=in-process&pivots=programming-language-javascript
 
 https://docs.microsoft.com/en-us/azure/azure-functions/functions-bindings-storage-queue-trigger?tabs=in-process%2Cextensionv5&pivots=programming-language-javascript